staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -75,14 +79,14 @@ type Event struct {
 	// API object from the request, in JSON format. The RequestObject is recorded as-is in the request
 	// (possibly re-encoded as JSON), prior to version conversion, defaulting, admission or
 	// merging. It is an external versioned object type, and may not be a valid object on its own.
-	// Omitted for non-resource requests.  Only logged at RequestObject Level and higher.
+	// Omitted for non-resource requests.  Only logged at Request Level and higher.
 	// +optional
-	RequestBody string
+	RequestObject runtime.Unknown
```

Comment:
> This is also true for protobuf, i.e. if the Event is serialized as protobuf, will the runtime.Unknown come through correctly, i.e. either as JSON or as some protobuf encoding of JSON?

`runtime.Unknown` should come through as its native encoding.  So the event fields would be protobuf, but this element would be json inside of it.  We use this for `Template.templates.openshift.io` which have an outer proto encoding with an inner json one.